src/main/java/org/myrobotlab/service/Keyboard.java

```diff
@@ -103,7 +103,7 @@ public void nativeMouseWheelMoved(NativeMouseWheelEvent me) {
   public Keyboard(String n, String id) {
     super(n, id);
     if (Service.isHeadless()) {
-      log.warn("the Keyboard service requires a DISPLAY to function correctly");
+      log.info("the Keyboard service requires a DISPLAY to function correctly");

```

Comment:
I think warn was fine here, info would be easy to miss

Reply:
I was not happy with it pumping out warnings when under ci testing .. but, with the new surefire reporting, it adds std out to the test that failed.  So scanning through the console with non-relevant warnings isn't as problematic.
I'll revert.